apps/whale-api/src/module.api/consortium.controller.ts

```diff
@@ -23,4 +23,18 @@ export class ConsortiumController {
       ttl: 600 // 10 minutes
     }) as AssetBreakdownInfo[]
   }
+
+  /**
+   *  Gets the mint stats information of a specific consortium member
+   *
+   * @return {Promise<MemberMintStatsInfo>}
+    */
+  @Get('/mintstats/:memberid')
```

Comment:
Would this be better to just be stats? So it can contain mint and burn stats in the future?

Reply:
Updated!